### PR TITLE
Version Conversion from opset 8 to 9

### PR DESCRIPTION
**Pull request by KsenijaS**

### CODE REVIEW

**Comment by spandantiwari** [comment]
cc: @houseroad @linkerzhang

**Comment by KsenijaS** [comment]
cc @bddppq @houseroad Can you please take a look at this PR? Thanks.

**Comment by houseroad** [comment]
Some type annotation is missing. Could you take care of it?

**Comment by KsenijaS** [comment]
@houseroad Can you please elaborate on what type annotations are missing?

**Comment by KsenijaS** [comment]
@houseroad I added type annotations and all checks pass now.

**Comment by vinitra** [comment]
@KsenijaS please merge the master branch into your branch! Otherwise LGTM as well.

**Comment by KsenijaS** [comment]
cc @houseroad @bddppq

File: onnx/test/version_converter_test.py [hunk 1]
```diff
@@ -730,6 +972,71 @@ def test_upsample_with_raw_initializer_9_8(self):  # type: () -> None
...
+        )]
+
+        graph = helper.make_graph(
+            nodes,
+            "test_cast",
+            [onnx.helper.make_tensor_value_info("X", data_type_from, [2, 3])],
+            [onnx.helper.make_tensor_value_info("Y", data_type_to, [2, 3])])
+
+        converted_model = self._converted(graph, helper.make_operatorsetid("", from_opset), to_opset)
+
```
Comment (houseroad) [inline onnx/test/version_converter_test.py]: check the node type?

**Pull request merged**